src/View.elm

```diff
@@ -239,7 +240,7 @@ accountTimelineView account statuses label iconName =
 statusActionsView : Mastodon.Status -> Html Msg
 statusActionsView status =
     let
-        target =
+        tootTarget =
```

Comment:
I think the general consensus in the code has been to use `status` for statuses, not `toot`. While I think `target` was fine, if you still want to precise its nature let's use `targetStatus` instead, for the sake of consistency.